packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -54,6 +54,7 @@ import { getRejectOnNotFound } from './utils/rejectOnNotFound'
 import { serializeRawParameters } from './utils/serializeRawParameters'
 import { validatePrismaClientOptions } from './utils/validatePrismaClientOptions'
 
+const sqltag = sqlTemplateTag.default
```

Comment:
Maybe `import sqltag, { Sql } from 'sql-template-tag'` instead on line 28 and replace `sqlTemplateTag.Sql` with just `Sql` in the file

Reply:
Good idea! Updated.